llama_hub/sec_filings/prepline_sec_filings/fetch.py

```diff
@@ -26,20 +25,16 @@ def inner(func):
 
     limits = fake_decorator
     sleep_and_retry = fake_decorator
-try:
-    from llama_hub.sec_filings.prepline_sec_filings.sec_document import (
-        VALID_FILING_TYPES,
-    )
-except ImportError:
-    from prepline_sec_filings.sec_document import VALID_FILING_TYPES
+
+from llama_hub.sec_filings.prepline_sec_filings.sec_document import VALID_FILING_TYPES
 
 SEC_ARCHIVE_URL: Final[str] = "https://www.sec.gov/Archives/edgar/data"
 SEC_SEARCH_URL: Final[str] = "http://www.sec.gov/cgi-bin/browse-edgar"
 SEC_SUBMISSIONS_URL = "https://data.sec.gov/submissions"
 
 
 def get_filing(
-    cik: Union[str, int], accession_number: Union[str, int], company: str, email: str
+    accession_number: Union[str, int], cik: Union[str, int], company: str, email: str

```

Comment:
why did you switch the arg positions?

Reply:
The text extraction from SEC documents is a demanding process, hence I implemented a multiprocessing method so that it can be faster. In the `secData.py` file, I have implemented parallel processing using a partial function

```python
get_filing_partial = partial(
        get_filing,
        cik=rgld_cik,
        company="Unstructured Technologies",
        email="support@unstructured.io",
    )

sec_extractor = SECExtractor(ticker=ticker)
```

For the partial function to work, the first argument needs to be the accession number (a unique identifier for each file). Hence, I switched the arguments. Is there a better way to do it?

Reply:
i see. i'm mostly trying to minimize the number of breaking changes, and seems like there's not a way to prevent this

Reply:
Yes, understood

It is not an user-facing function, hopefully it will break previous implementations.